src/types/storage.ts

```diff
@@ -1,27 +1,29 @@
+import { NotificationTriggers } from "../constants";
+
 export interface Withsha {
   sha?: string;
 }
 
 export type StorageUser = {
-  telegramId: number;
-  githubId: number;
-  githubUsername: string;
+  telegram_id: number;
```

Comment:
Decided to use snake_case for the database items because it's easier to read in that env, lmk if this is strictly not allowed (I know we prefer camelCase)

Reply:
> it's easier to read in that env

What do you mean? 

The only exceptions I can think of are external libraries and perhaps table column headers.

Reply:
> table column headers

that's exactly what I mean, reading SB table headers in camelCase ruins me. As GitHub storage it's a JSON object so it's much easier with camelCase which is why I'm having to switch back again.

I'll use whatever you think is best I knew this might not be the best idea but you don't know til you try.